joschahenningsen/TUM-Live
# Moderate replies

### Motivation and Context
- Resolve #780 

### Description
Add TS logic to handle reply moderation

### Steps for Testing
- 2 Account (1 Admin, 1 Studi/etc)
- 1 Livestream with moderated chat

1. Log in
2. Navigate to a Livestream
3. Reply to a message with the `Studi` account 
4. Approve reply with the `Admin` account
5. Message should now be visible for all.
